packages/subgraph/src/schema.graphql

```diff
@@ -0,0 +1,17 @@
+type Greeting @entity {
+  id: ID!
```

Comment:
Would be great to follow recent best practices. Like "Bytes as IDs" as described here: https://medium.com/edge-node-engineering/two-simple-subgraph-performance-improvements-a76c6b3e7eac